.github/workflows/push.yml

```
@@ -64,28 +70,16 @@ jobs:
           client-payload: '{"branch": "${{ steps.extract_branch.outputs.branch}}" }'
```

if you remove the step extract_branch from this job this line needs to change to an needs syntax 
```suggestion
          client-payload: '{"branch": "${{ needs.branch_name.outputs.branch }}" }'
```

---

nice, https://docs.github.com/en/actions/learn-github-actions/workflow-syntax-for-github-actions#jobsjob_idoutputs